internal/cmd/commands/dev/dev.go

```diff
@@ -565,7 +568,23 @@ func (c *Command) Run(args []string) int {
 	for !shutdownTriggered {
 		select {
 		case <-c.ShutdownCh:
-			c.UI.Output("==> Boundary dev environment shutdown triggered")
+			c.UI.Output("==> Boundary dev environment shutdown triggered, interrupt again to force")
+
+			// Add a force-shutdown goroutine to consume another interrupt
+			abortForceShutdownCh := make(chan struct{})
+			defer close(abortForceShutdownCh)
+			go func() {
+				shutdownCh := make(chan os.Signal, 4)
+				signal.Notify(shutdownCh, os.Interrupt, syscall.SIGTERM)
```

Comment:
I think this is missable if the second interrupt arrives prior to this being registered. I don't have a great idea for how to do it better, and it's unlikely to happen since the second would have to come _very_ quickly. Just calling it out in case you have other ideas.

Reply:
Yeah, none short of rewriting [`MakeShutdownCh`](https://github.com/hashicorp/boundary/blob/4ba7eb5ff44d9a1541adf42f70d85541ed82ce28/internal/cmd/base/base.go#L106-L119). I'll noodle on it for a few days.

Reply:
Resolving for now, I don't have a better idea and I think it's a very unlikely scenario.